apm-agent-plugins/apm-spring-webmvc/apm-spring-webmvc-spring5/src/test/java/co/elastic/apm/agent/springwebmvc/exception/AbstractExceptionHandlerInstrumentationWithExceptionResolverTest.java

```diff
@@ -39,8 +43,12 @@
 })
 public abstract class AbstractExceptionHandlerInstrumentationWithExceptionResolverTest extends AbstractExceptionHandlerInstrumentationTest {
 
-    @Test
-    public void testCallApiWithExceptionThrown() throws Exception {
+    @ParameterizedTest
+    @ValueSource(booleans = {true,false})
+    public void testCallApiWithExceptionThrown(boolean useAttributeBasedPropagation) throws Exception {
+        CoreConfiguration coreConfig = config.getConfig(CoreConfiguration.class);
+        doReturn(useAttributeBasedPropagation).when(coreConfig).isNotUseServletAttributesForExceptionPropagation();
```

Comment:
little inconsistency between the variable name and the currently negative config option name (but fine as long as we test both in practice).